.fvmrc

```diff
@@ -0,0 +1,4 @@
+{
+  "flutter": "3.19.0",
+  "updateVscodeSettings": false
```

Comment:
Why?

Reply:
Because it will set the sdk path to the version and not the symlink. If we don't block it then we will always have to manually edit the flutter update PRs which I really dislike.
Proper solution would be to fix this in fvm because I don't think this is intended, at least it worked differently before.

Reply:
A big benefit of fvm 3.0 is that is that it will auto map the `flutter` path to the desired location.
This means that new devs (using the vscode terminal) would only need to run `flutter $command` instead of `fvm flutter $command` lowering the barrier of entry.

Looking back to previous contributors I think that many got confused by the use of fvm so we should really consider it.
Yes one can argue that it's the contributors fault not having read (and understood) our contributing guidlines. But on the other hand we already require quite a lot of knowledge (melos, conventional commits, fvm, code generation) and we should try to also include devs not yet familiar with these topics.

<details>
<summary>Without `updateVscodeSettings`</summary>
➜  nextcloud-neon git:(renovate/fvm-3.x) ✗ fvm use master
➜  nextcloud-neon git:(renovate/fvm-3.x) ✗ fvm flutter --version
[WARN] FVM_HOME environment variable is deprecated. 
Please use FVM_CACHE_PATH instead

Flutter 3.20.0-4.0.pre.38 • channel master • https://github.com/flutter/flutter.git
Framework • revision 92c7ba8191 (vor 10 Tagen) • 2024-02-08 03:21:52 -0500
Engine • revision 6d524588c0
Tools • Dart 3.4.0 (build 3.4.0-116.0.dev) • DevTools 2.32.0

➜  nextcloud-neon git:(renovate/fvm-3.x) ✗ flutter --version    
Flutter 3.19.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision bae5e49bc2 (vor 4 Tagen) • 2024-02-13 17:46:18 -0800
Engine • revision 04817c99c9
Tools • Dart 3.3.0 • DevTools 2.31.1
➜  nextcloud-neon git:(renovate/fvm-3.x) ✗
</details>

<details>
<summary>With `updateVscodeSettings`</summary>
➜  nextcloud-neon git:(renovate/fvm-3.x) ✗ fvm use master
➜  nextcloud-neon git:(renovate/fvm-3.x) ✗ fvm flutter --version
[WARN] FVM_HOME environment variable is deprecated. 
Please use FVM_CACHE_PATH instead

Flutter 3.20.0-4.0.pre.38 • channel master • https://github.com/flutter/flutter.git
Framework • revision 92c7ba8191 (vor 10 Tagen) • 2024-02-08 03:21:52 -0500
Engine • revision 6d524588c0
Tools • Dart 3.4.0 (build 3.4.0-116.0.dev) • DevTools 2.32.0

➜  nextcloud-neon git:(renovate/fvm-3.x) ✗ flutter --version   
Flutter 3.20.0-4.0.pre.38 • channel master • https://github.com/flutter/flutter.git
Framework • revision 92c7ba8191 (vor 10 Tagen) • 2024-02-08 03:21:52 -0500
Engine • revision 6d524588c0
Tools • Dart 3.4.0 (build 3.4.0-116.0.dev) • DevTools 2.32.0
</details>

Reply:
I'm not sure I understand what you are saying. If the path just points to the symlink, then it should work the exact same way, no?

Reply:
Let's merge it for now and I'll investigate this further.
For some reason